index.html

```diff
@@ -0,0 +1,389 @@
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/css?family=Alfa+Slab+One|Open+Sans&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Fira+Sans+Condensed&display=swap" rel="stylesheet">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+     <link rel="stylesheet" href="index.css">
+    <title>NEWSWEEK</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+  </head>
+  <body>
+
+
+<hr>
+<nav id="up">
+  <date class="d">Thu,May 23,2019</date>
+<div class="logo">
+<h4 class="hi"> NEWSWEEK</h4>
+</div>
+<span class= "flow">
+<form class="form-inline">
+    <button class="btn btn-outline btn0" type="button" >SIGN IN</button>
+    <button class="btn  btn-outline btn1" type="button">SUBSCRIBE > </button>
+  </form>
+</span>
+</nav>
+
+
+<nav>
+<ul class="navlinks">
+
+<li>
+  <a href ="/world" >
+U.S.
+  </a>
+</li>
+
+
+<li>
+  <a href ="/business" >
+    World
+
+</a>
+</li>
+
+
+<li>
+  <a href ="/tech-science">
+    Business
+
+  </a>
+</li>
+
+
+<li>
+  <a href ="/culture" >
+    Tech & Science
+  </a>
+</li>
+
+
+<li>
+  <a href ="/sports" >
+    Culture
+  </a>
+</li>
+
+<li>
+  <a href ="/health" >
+    Sports
+  </a>
+</li>
+
+<li>
+  <a href ="/opinion" >
+    Health
+  </a>
+</li>
+
+<li>
+  <a href ="https://www.newsweek.com/ndelible">
+    Opinion
+  </a>
+</li>
+
+<li>
+  <a href ="#">
+  Ndelible
+   <i class="fas fa-search"></i>
+  </a>
+</li>
+
+</ul>
+<div class="burger">
+  <div class ="line1"></div>
+  <div class ="line2"></div>
+  <div class ="line3"></div>
+
+</div>
+</nav>
+
+
+
+<hr>
+
+
+
+
+<!--- main body begains      -->
+<div class="container my-container" >
+<div class="row  my-row">
+  <div class="col-md-4  my-col" width="100%">
+    <div class ="l">
+        <img src="trp.png" width="100%">

```

Comment:
Edit all src property they are not pointing to the right location for resource, use this `src="./img/trp.png"`